src/main/java/seedu/address/ui/ItemCard.java

```diff
@@ -47,9 +51,20 @@
     public ItemCard(Item item, int displayedIndex) {
         super(FXML);
         this.item = item;
+        String itemQuantity = item.getQuantity().value;
+        String itemMaxQuantity = item.getMaxQuantity().value;
+
         id.setText(displayedIndex + ". ");
         name.setText(item.getName().fullName);
-        quantity.setText(item.getQuantity().value);
+        quantity.setText(itemQuantity);
+        if (Integer.parseInt(itemMaxQuantity) > 0) {
+            float percentage = Float.parseFloat(itemQuantity) / Integer.parseInt(itemMaxQuantity) * 100;
+            String stats = String.format("/%s (%.01f%%)", itemMaxQuantity, percentage);
+            quantityStats.setText(stats);
+            if (percentage > 100) {
+                quantityStats.setFill(Color.RED);
+            }
+        }
```

Comment:
Similarly if maxQuantity can now be empty i think having a method to check if maxQuantity is null prior to this block of code would prevent a parsing error